From 87ee4bbe5d875ae41d349395b101166dc82eeac9 Mon Sep 17 00:00:00 2001
From: Wang Jie <dave.wang@rock-chips.com>
Date: Fri, 28 Feb 2020 13:01:41 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368: modify vcc_lcd configure for
 rk3368 808 evb

1.When the system starts, enable vcc_lcd.
2.when the System sleep, disable vcc_lcd.

Change-Id: I29afc3e98a8cbe7ded65a8b9de1990fa2ea26b53
Signed-off-by: Wang Jie <dave.wang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-808.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi
index 281bf7bf31d6..8a7941652a7d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi
@@ -85,6 +85,10 @@
 		pinctrl-0 = <&lcd_pwr>;
 		regulator-name = "vcc_lcd";
 		enable-active-high;
+		regulator-boot-on;
+		regulator-state-mem {
+			regulator-off-in-suspend;
+		};
 	};
 
 	vcc_otg_vbus: otg-vbus-regulator {
-- 
2.35.3

